Documentation/reporting_bugs.md

```diff
@@ -1,6 +1,6 @@
 # Reporting Bugs
 
-If you find bugs or documentation mistakes in etcd project, please let us know by [opening an issue](https://github.com/coreos/etcd/issues/new). We treat bugs and mistakes very seriously and believe no issue is too small. Before creating a bug report, please check there that one does not already exist.
+If you find bugs or documentation mistakes in etcd project, please let us know by [opening an issue][issue]. We treat bugs and mistakes very seriously and believe no issue is too small. Before creating a bug report, please check there that one does not already exist.
```

Comment:
`s/in etcd project/in the etcd project/`
`s/there that/that/`
`s/one/an issue reporting the same problem/`